libgalois/include/katana/PropertyGraph.h

```diff
@@ -360,11 +360,15 @@ class KATANA_EXPORT PropertyGraph {
 
   PropertyGraph(
       katana::GraphTopology&& topo_to_assign,
-      NUMAArray<EntityTypeID> node_entity_type_id,
-      NUMAArray<EntityTypeID> edge_entity_type_id) noexcept
+      NUMAArray<EntityTypeID>&& node_entity_type_id,
```

Comment:
Are we following the model that the type array is kept separate from the manager? Why can't it be a part of the manager?